applications/openshift/master/file_groupowner_master_admin_kubeconfigs/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Verify Group Who Owns The OpenShift Admin Kubeconfig Files'
+
+description: |-
+  {{{ describe_file_group_owner(file="/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/.*\.kubeconfig", group="root") }}}
+
+rationale: |-
+  There are various kubeconfig files that can be used by the administrator,
+  defining various settings for the administration of the cluster; They also
+  contain credentials that could be used to control the cluster. They are
+  needed for disaster recovery and each points to a different endpoint in the
+  cluster. You should restrict its file permissions to maintain the integrity
```

Comment:
```suggestion
  There are various kubeconfig files that can be used by the administrators
  to define various settings for the administration of the cluster. These files
  contain credentials that can be used to control the cluster and are
  needed for disaster recovery and each kubeconfig points to a different endpoint in the
  cluster. You should restrict its file permissions to maintain the integrity of the kubeconfig
  file as an attacker who gains access to these files can take over the cluster.
```

Reply:
The last sentence is important when writing rationale's as it is needed to help people understand why this rule exists rather than just creating a rule to create a rule.